utils/db-parts.stanza

```diff
@@ -85,7 +85,7 @@ public defstruct Resistor <: Component :
   TCR: TCR|False ; Temperature coefficient of resistance (ohms/ohm*degC)
   sellers: Tuple<Seller>|False with: (as-method => true)
   resolved-price: Double|False with: (as-method => true)
-  component: ComponentCode|False
```

Comment:
Ok, I read the explanation in comment later on

Reply:
> Nice, I am wondering what is in the`ComponentCode` of those v1 parts. I assume you are currently reconstructing what should be in it and it is a work in progress?

I think you already saw this, but just in case others are wondering...

For V1 parts, we set the `ComponentCode`, but some fields are left empty (e.g. pins/symbols/landpattern).

We conditionally then insert those fields in the OCDB code - for example:
- [defmethod to-jitx (resistor: Resistor) -> Instantiable :](https://github.com/JITx-Inc/open-components-database/blob/e5cfcb390a5c7d5b6325d23615575d30b272b1b4/utils/db-parts.stanza#L151)

There's a task to move this logic offline: JITX-4198

But, we're being careful about moving that over, as we want to make sure it's done correctly (e.g. "spice" isn't currently part of the parts-db schema).

So in the short-term, we'll stick with the conditional client logic - but, the parts are at least sharing the same backend DB.

Reply:
> We conditionally then insert those fields in the OCDB code

My observation is that for v1, we are not using any data in `ComponentCode` yet. So I assume other changes are coming to have `to-jitx` use this data

Reply:
Ah, yes - the work for JITX-4198 would include removing the fallback paths entirely - correct!